changelog/std_string.dd

```diff
@@ -0,0 +1,3 @@
+Added `core.stdcpp.string`.
+
+Added `core.stdcpp.string`, which links against C++ `std::string`
```

Comment:
Should have stated that it's only for Windows :/

Reply:
Good point. I'll update that, or I'll just make the other platforms CI work. (Well, Linux still has the interior point issue...)